django_tutorial/first_project/web_app/static/web_app/js/index.js

```diff
@@ -0,0 +1,385 @@
+// TODO: add browserify or webpack to build pipeline, and break this into separate js files with imports,
```

Comment:
I think I'm just bad at reading javascript. Where in this are you pointing it to our database (or where is the data coming from here?)